crates/nu-cli/src/repl.rs

```diff
@@ -188,6 +189,18 @@ pub fn evaluate_repl(
         );
     }
 
+    if engine_state.get_config().show_startup_time && !engine_state.get_config().show_banner {
+        let dd = Color::Default.dimmed().prefix().to_string();
+        println!(
+            "{dd}startup time: {}",
+            Value::Duration {
+                val: engine_state.get_startup_time(),
+                span: Span::unknown()
+            }
+            .into_string("", engine_state.get_config())
```

Comment:
This is kind of a separate thought, but looking through this `evaluate_repl()` function a bit more. I'm wondering now if it there is a performance impact to calling `engine_state.get_config()` 9 times (if i counted correctly). If would be nice to grab the config just once, if we can. There may be some reason we call it multiple times. I haven't looked that close.